applications/rag/metadata.yaml

```diff
@@ -17,27 +17,15 @@ spec:
   content: {}
   interfaces:
     variables:
-      - name: goog_cm_deployment_name
-        description: The name of the deployment.
-        varType: string
-        required: true
-      - name: add_auth
-        description: Enable iap authentication on jupyterhub
-        varType: bool
-        defaultValue: false
       - name: autopilot_cluster
         varType: string
-        defaultValue: "false"
-      - name: brand
-        description: name of the brand if there isn't already on the project. If there is already a brand for your project, please leave it blank and empty
-        varType: string
-        defaultValue: ""
-      - name: client_id
-        description: Client ID used for enabling IAP
+        defaultValue: false
+      - name: cloudsql_instance
```

Comment:
is it possible to prefix the instance name and bucket name with the IM deployment name? like we do for the other resources (like SAs)?

Reply:
will do in next PR.